### DISCUSSION TITLE
Entities can be removed?

### DISCUSSION THREAD

**Discussion by rleongcs** [Blackymas/NSPanel_HA_Blueprint#779] [Q&A] [answered]
With the recent update to ver 3.3, I see there are 7 entities became "unavailable". Just wanted to confirm that I can remove them.

I did see from the v3.3 release notes about 2 entitites which I know could be removed but there are 5 more that I wanted to confirm.
```
16. Sleep modus" renamed to "Sleep mode", by @edwardtfn (#714)
17. Switch "Nextion inited" is now "Nextion display" sensor, by @edwardtfn (#715)
```

![image](https://user-images.githubusercontent.com/61628312/236138073-9fc578b4-e8b0-41ee-b0f6-e49bd3f39eed.png)

Can these 5 be removed?
![image](https://user-images.githubusercontent.com/61628312/236138821-1c12b669-61ad-4143-85a8-79fbdbe7b941.png)

**Comment by edwardtfn** [comment, marked as answer]
You can safely remove all of those.

The new `sensor.xxxxx_nspanel_event` removed the need of all these `Last click xxx` and `Currentpage`.
`Sleep modus` was replaced by `Sleep mode`, with similar behaviour.
`Nextion inited` was replaced by a sensor called `Nextion display` which shows the status of communication between the display and ESP32 (both inside your panel).

**Reply by rleongcs** [reply]
Have a little problem to delete as the "DELETE" button is grayed out. How do I go about deleting those entities?
![image](https://user-images.githubusercontent.com/61628312/236269245-7c138f2e-ce86-4e55-ba24-25ce6dc7d03e.png)
![image](https://user-images.githubusercontent.com/61628312/236269425-5b79e037-123f-491a-a155-dcca93298973.png)

**Reply by rleongcs** [reply]
Just to update that I managed to delete the said entities after a restart of HA which enables the "DELETE" button. 🎉
